tough/tests/repo_editor.rs

```diff
@@ -63,9 +121,510 @@ fn repository_editor_from_repository() {
     assert!(RepositoryEditor::from_repo(&root, repo).is_ok());
 }
 
+//Create sign write and reload repo
```

Comment:
All existing comments have a space after the slashes. New comments should as well.

```suggestion
// Create sign write and reload repo
```